nextcord/abc.py

```diff
@@ -1367,10 +1367,15 @@ async def send(
         if embed is not None and embeds is not None:
             raise InvalidArgument("Cannot pass both embed and embeds parameter to send()")
 
+        local_embed_files: List[File] = []
```

Comment:
After this line, you should set the `embeds` list to a list that only contains `embed` if `embed` is set
This should eliminate the repetitive code below